src/js/contact.js

```diff
@@ -0,0 +1,22 @@
+const contactContent = (content) => {
+  const contactTitle = document.createElement('h4');
+  contactTitle.textContent = '+27 7875 090 123';
+  contactTitle.setAttribute('class', 'contact-title');
+  content.appendChild(contactTitle);
+  const divEl = document.createElement('div');
+  // divEl.setAttribute('class', 'contact-background');
+  divEl.setAttribute('class', 'background');
```

Comment:
It’s better to delete **commented pieces of code**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))